src/main/java/com/litequizapp/service/AnswerRestService.java

```diff
@@ -39,29 +40,39 @@ public AnswerEntity getAnswerById(long id) {
 
   }
 
-  public AnswerEntity createAnswer(AnswerEntity answerEntity) {
-    if (answerEntity.getTitle() == null){
+
+  public AnswerDTO createAnswer(AnswerDTO answerDTO) {
+    if (answerDTO.getTitle() == null || answerDTO.getQuestionId() == null) {
```

Comment:
I'd suggest to use this article https://habr.com/ru/post/343960/ to get rid of this ugly check.

Reply:
If I refuse this validation and do like in the article I couldn't throw exceptions with my own messages, isn't so?

Reply:
Done https://github.com/sshitov/liteQuizApp/pull/8/commits/ead791a5547251884a48b20d052911035e9c9a9f